src/test/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceServiceSpec.scala

```diff
@@ -137,6 +158,10 @@ class ResourceServiceSpec
   private val constrainableService =
     new ResourceService(constrainableResourceTypes, constrainablePolicyEvaluatorService, policyDAO, dirDAO, NoExtensions, emailDomain, Set.empty)
 
+  val mockAzureService = mock[AzureService]
+  private val azuredService =
```

Comment:
is this a typo should this be azureService? Also isnt there a AzureService class separate from the ResourceService class? Should this val be 'resourceService' instead?

Reply:
renamed for clarity.